app/src/main/java/org/mozilla/fenix/components/PrivateShortcutCreateManager.kt

```diff
@@ -38,22 +46,35 @@ object PrivateShortcutCreateManager {
                 )
             )
             .setIcon(icon)
-            .setIntent(Intent(context, HomeActivity::class.java).apply {
-                action = Intent.ACTION_VIEW
-                flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
-                putExtra(HomeActivity.PRIVATE_BROWSING_MODE, true)
-                putExtra(
-                    HomeActivity.OPEN_TO_SEARCH,
-                    StartSearchIntentProcessor.PRIVATE_BROWSING_PINNED_SHORTCUT
-                )
-            })
+            .setIntent(intent)
             .build()
+
+
+        return shortcut
+    }
+
+    fun getIcon(context: Context): IconCompat {
```

Comment:
I would probably avoid these one line function since it's not being reused.

Reply:
removed